_includes/help/nav_sidenav.html

```diff
@@ -1,20 +1,44 @@
-<nav class="sm-col-right sm-col-4 sm-show">
-  <ul class="list-reset pt2">
-    {% for question in related_questions %}
-      {% if question.url == page.url %}
-        {% assign _index = forloop.index | modulo:7 %}
-        {% if _index == 0 %}
-          {% assign _index = 1 %}
+<div class="grid-col grid-col-3 grid-offset-2">
+  {% include search.html size="small" %}
+  <nav class="margin-y-4 sidenav">
+    <ul class="usa-accordion usa-sidenav">
+      {% for subpage in site.help_pages %}
+      <li class="usa-sidenav__item__empty">
+        {% assign subpage_slug = subpage | slugify %}
+        {% assign related_questions = site.help | where_exp: "item", "item.url contains subpage_slug" | sort: 'order' %}
+        {% capture is_initially_expanded %}
+        {% if page.url contains subpage_slug %}true{% else %}false
         {% endif %}
-      {% endif %}
-      <a class="block h6 nav-sidenav-item {% if question.url == page.url %}active{% endif %}" href="{{ question.url | prepend: site.baseurl }}">
-        <li class="p2 {% unless forloop.last %}border-bottom{% endunless %}">
-          {% assign url_size = question.url | size | minus: 1 %}
-          {% assign page_title = question.url | replace: "/", "." | slice: 1, url_size %}
-          {% capture page_title %}{% t page_title %}{% endcapture %}
-          {{ page_title | markdownify }}
-        </li>
-      </a>
-    {% endfor %}
-  </ul>
-</nav>
+        {% endcapture %}
+        {% capture is_initially_hidden %}
+        {% if page.url contains subpage_slug %}false{% else %}true
+        {% endif %}
+        {% endcapture %}
+      <li class="usa-sidenav__item usa-parent {% if is_initially_expanded %}usa-current{% endif %}"
+        aria-expanded="{{ is_initially_expanded }}">
+        {{ site["translations"][site.lang]["help_subpages"][subpage] }}
+      </li>
+
+      <div aria-hidden="{{ is_initially_hidden }}">
```

Comment:
While I see we're assigning `aria-hidden` for the inactive section, I can still visually see the links.

![image](https://user-images.githubusercontent.com/1779930/102495259-e1c98900-4043-11eb-9ed7-f528a9078bda.png)